### PR TITLE
Frontend -- Create custom toggle component

### PR DESCRIPTION
**Pull request by AaronFmyHub**
Issue #103

# What type of PR is this? (Check all that apply)

- [x] ✨**feature**: Introduces completely new code or new features.
- [ ] 🐛**fix**: Implements changes that fix a bug. Ideally, reference an issue if present.
- [ ] ♻️**refactor**: Includes any code-related change that is neither a fix nor a feature.
- [ ] ✅**build**: Encompasses all changes related to the build of the software, including changes to dependencies or the addition of new ones.
- [ ] ⚡️**test**: Pertains to all changes regarding tests, whether adding new tests or modifying existing ones.
- [ ] 🚰**ci**: Involves all changes related to the configuration of continuous integration, such as GitHub Actions or other CI systems.
- [ ] 📚**docs**: Includes all changes to documentation, such as README files, or any other documentation present in the repository.
- [ ] 🗑️**chore**: Captures all changes to the repository that do not fit into the above categories.

# Description

Add toggle using react switch under ~/components

# Tests

How was this tested?

- [ ] Unit tests
- [ ] Integration tests
- [ ] E2E tests
- [ ] Manual tests
- [ ] Tests were NOT needed
- [x] Other (explain below)

Explanation:
Default Off - 
Screenshot 2024-03-24 185809
Click - 
Screenshot 2024-03-24 190208

# Documentation

- [ ] Added to README.me
- [ ] Seperate document
- [x] NO documentation needed

Link to external documentation:

### CODE REVIEW

File: apps/expo/src/components/toggle/Toggle.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    const toggleSwitch = () => {
+        setIsEnabled(previousState => !previousState);
+    };
+
+    return (
+        <SafeAreaView style={{backgroundColor: "#1C1B1B", flex: 1}}>
+            <View className="items-center" >
+                <Switch
+                    trackColor={{ false: "#CACACA", true: "#67D2A5" }}
+                    thumbColor={isEnabled ? "#CACACA" : "#CACACA"}
```
[the remark this answers is not in the document]
Comment (thienguen) [reply apps/expo/src/components/toggle/Toggle.tsx:17]: https://snack.expo.dev/@thienguen/7a865c

**Comment by thienguen** [comment]
conversation added if you want to go the custom route, OR you can install a library from react-ios-switch.

from [here](https://www.npmjs.com/package/react-ios-switch)

Steps:

[ ] navigate to apps/expo (the folder)
[ ] run pnpm i react-ios-switch
[ ] follow the example in the link
[ ] profit

File: apps/expo/src/components/toggle/Toggle.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (thienguen) [reply apps/expo/src/components/toggle/Toggle.tsx:17]: @AaronFmyHub this is the custom switch route

[the remark this answers is not in the document]
Comment (AaronFmyHub) [reply apps/expo/src/components/toggle/Toggle.tsx]: image

[Screen_recording_20240326_024547.webm](https://github.com/UNLV-CS472-672/2024-S-GROUP3-Barbell/assets/146596967/cc60f776-78b4-4b24-81ea-e06bed2cafd1)

**Review by thienguen — APPROVED** [review]

**Pull request merged**